ste/downloader-azureFiles.go

```diff
@@ -78,14 +78,26 @@ func (bd *azureFilesDownloader) Epilogue() {
 			err := spdl.PutFileSMBProperties(bd.sip.(ISMBPropertyBearingSourceInfoProvider), bd.txInfo)
 
```

Comment:
Perhaps we should change this to PutDestinationSMBProperties, if it's going to be re-used for folders.

Reply:
Is it "File" as in Azure files, or "file" as in thing-that's-not-a-folder ;-)  I don't know!

Reply:
I've renamed it to PutSMBProperties. That seemed consistent with the other method in the interface:  PutSDDL

Reply:
This sounds good.

---
In reply to: [391993472](https://github.com/Azure/azure-storage-azcopy/pull/898#discussion_r391993472) [](ancestors = 391993472)